project/CMakeLists.txt

```diff
@@ -281,11 +287,12 @@ maybe_add_dependencies(fizz folly)
 maybe_add_dependencies(wangle folly fizz)
 maybe_add_dependencies(fbthrift folly krb5 bison flex mstch zlib zstd wangle fatal)
 maybe_add_dependencies(proxygen wangle libunwind gperf)
+maybe_add_dependencies(librdkafka berkeleydb libcurl libtool openssl cyrus-sasl)
```

Comment:
Maybe move this line after line 294 
so that `librdkafka` is only compiled when `ENABLE_ROCKSDB_CLOUD` is true.  WDYT?

Reply:
good catch